multigeiger/loraWan.cpp

```diff
@@ -296,11 +309,10 @@ transmitionStatus_t lorawan_send(uint8_t txPort, uint8_t * txBuffer, uint8_t txS
                 return txStatus;
           }
```

Comment:
sketch/loraWan.cpp: In function 'transmissionStatus_t lorawan_send(uint8_t, uint8_t*, uint8_t, bool, uint8_t*, uint8_t*, uint8_t*)':
loraWan.cpp:299:18: error: enumeration value 'TX_STATUS_ENDING_ERROR' not handled in switch [-Werror=switch]
           switch ( txStatus ) {

why am I suddenly seeing this? i did not change this code block.

how should it be handled?

Reply:
I don't know - here it is only a warning !

multigeiger/loraWan.cpp: In function 'transmissionStatus_t lorawan_send(uint8_t, uint8_t*, uint8_t, bool, uint8_t*, uint8_t*, uint8_t*)':
multigeiger/loraWan.cpp:299:18: warning: enumeration value 'TX_STATUS_ENDING_ERROR' not handled in switch [-Wswitch]
           switch ( txStatus ) {
                  ^
multigeiger/loraWan.cpp:299:18: warning: enumeration value 'TX_STATUS_TIMEOUT' not handled in switch [-Wswitch]

You could change the -Werror=switch to -Wswitch

Reply:
fixed compiler warning without changing behaviour. should be checked nevertheless...

Reply:
Having "compiler warning: all" in Arduino IDE. Seems to transform some warnings to errors...